### PR TITLE
tls: add a test for certfile.c

### PR DESCRIPTION
**Pull request by allisonkarlitskaya**

### CODE REVIEW

File: src/tls/Makefile-tls.am [hunk 1]
```diff
@@ -52,14 +52,25 @@ TEST_LDADD = \
 	$(NULL)
 
 TLS_TESTS = \
-	test-tls-connection				\
-	test-tls-server					\
+	test-tls-connection \
+	test-tls-server \
+	test-certfile \
```
Comment (martinpitt) [inline src/tls/Makefile-tls.am]: Maybe test-tls-certfile for consistency?

Comment (allisonkarlitskaya) [reply src/tls/Makefile-tls.am]: It seemed redundant, but no strong feelings either way.

**Comment by allisonkarlitskaya** [comment]
src/tls/test-certfile.c:181:13: error: ignoring return value of ‘read’, declared with attribute warn_unused_result [-Werror=unused-result]
             read (connections[slot_nr], &b, sizeof b);
             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

tsk tsk tsk

File: src/tls/Makefile-tls.am (hunk 1 above)
Comment (allisonkarlitskaya) [reply src/tls/Makefile-tls.am]: Took your suggestion.

**Pull request merged**